Add 2019年对话式AI的10篇重要研究论文.md (#57)
opensourceai/spark-plan: article/2019/12/2019年对话式AI的10篇重要研究论文.md

本文建立在细化可能性模型的基础上，该模型认为，有说服力的信息根据人们的世界观而量身定制时会更有效。作者从Mechanical Turk招募了参与者，从心理上对他们进行了介绍，然后请他们扮演角色，说服对方，向慈善机构Save the Children捐款。然后，研究人员根据所使用的说服策略对会话的一部分进行注释，并建立了一个混合RCNN分类器，将整个语料库分为不同的说服策略。他们分析了总体上最成功的策略以及说服策略与用户的人口统计特征和性格类型之间的相互作用。

 

![会话式AI-说服数据集](img/3_Persuasion_dataset_500px_web.jpg)

 

#### **本文的核心思想是什么？**

-  AI对话智能体的目的通常涉及某种形式的说服，但是对他们有效的说服策略的研究受到限制，因为说服的研究传统上是社会科学而不是AI工程的一部分。 
- 这项跨学科的论文建立在社会学基础上，可以在大量的人类聊天对话中识别不同的说服策略，并分析哪种策略对一般人和给定的性格类型最有效。
- 为了将说服策略分为10类，再加上一类“非策略”类，作者开发了具有以下特征的混合RCNN模型：
  - 句子嵌入
  - 上下文嵌入
  - 句子级功能

 

![对话式AI-说服力](img/3_Persuastion_classifier_500px_web.jpg)

*混合RCNN分类器概述*

 

#### **关键成就是什么？**

- 研究人员发现：
  - 提供实用的捐赠信息是总体上最好的策略。
  - 向用户询问他们是否熟悉慈善机构，大大提高了对开放度五大特征的参与者的捐赠可能性。
  - 提出个人问题会大大提高订阅者对“关爱”或“自由”的捐赠概率（按照海特的道德基础理论），但会降低对订阅“公平或权威”的捐赠者的捐赠概率。

 

#### **人工智能界对此有何看法？**

- 该论文已被提名为ACL 2019最佳论文奖的候选人。

 

#### **未来的研究领域是什么？**

- 通过包含更多注释和更多对话框上下文来提高分类器的性能。
- 设计一种自适应的说服对话系统，该系统具有根据用户个人资料选择说服策略的能力。

 

#### **有哪些可能的业务应用程序？**

- 作者指出，技术是双刃剑，并具有说服人们善与恶的潜力。牢记这一点，本文中介绍的知识可用于为自动对话系统的道德设计基础提供信息。
- 按照本文中的示例，慈善机构可以使用它说服潜在的支持者捐款。
- 它也可以用于旨在帮助人们实现其个人目标的产品中。

 

#### **在哪里可以获得实现代码？**

- 作者已经在[GitLab上](https://gitlab.com/ucdavisnlp/persuasionforgood)发布了数据集和代码。

 

> ### **4.**  \4. [OPENDIALKG：基于注意力的知识图遍历的可解释性对话推理](https://www.semanticscholar.org/paper/OpenDialKG%3A-Explainable-Conversational-Reasoning-Moon-Shah/0d3c68c207fc83fb402b7217811af22066300fc9)，By SEUNGWHAN MOON，PARARTH SHAH，ANUJ KUMAR和RAJEN SUBBA 

Reviewer: ```suggestion
### 4.[OPENDIALKG：基于注意力的知识图遍历的可解释性对话推理](https://www.semanticscholar.org/paper/OpenDialKG%3A-Explainable-Conversational-Reasoning-Moon-Shah/0d3c68c207fc83fb402b7217811af22066300fc9)，By SEUNGWHAN MOON，PARARTH SHAH，ANUJ KUMAR和RAJEN SUBBA 
```

 

#### **原始摘要**

 我们研究了一种对话推理模型，该模型策略性地遍历了大型公共事实知识图（KG），以引入让人感兴趣且在上下文中不同的实体和属性。在这项研究中，我们收集了一个新的**Open-ended Dialog**↔ **KG**平行语料库，称为**OpenDialKG**，其中来自人与人角色扮演对话的15K的每个讲话都用地面真实引用手动注释，这些引用来自具有1M +事实的大型KG的相应实体和路径。然后，我们提出DialKG Walker模型，该模型学习对话上下文作为KG上的结构化遍历的对话上下文的符号转换，并预测自然实体，通过新颖的领域无关，基于注意力的图路径解码器引入给定的先前对话上下文。自动和人工评估表明，在域内和跨域任务中，我们的模型都可以比最新的基准或基于规则的模型检索更多自然和类似人类的响应。所提出的模型还为检索到的每个实体生成了KG行走路径，从而提供了一种解释对话推理的自然方法。 

 

#### **我们的总结**

Facebook对话式AI团队采用了一种新颖的方法来创建自然的，类似于人的反应，并提供有关不同实体及其属性的引人入胜的，上下文相关的信息。首先，他们收集一个新的并行语料库**OpenDialKG**，在对话中对实体的每次提及都与其相应的真实KG路径手动链接。然后，作者介绍了一种称为**DialKG Walker**的新模型，该模型可以学习对话中提到的实体之间的知识路径以及基于大型常识知识图的原因。在多个基准上进行的实验表明，与最新的基准相比，建议的方法可产生更多自然和类似人的反应。

 

![OpenDialKG对话式AI](img/4_OpenDialKG_overview_550px_web.jpg)

 

#### **本文的核心思想是什么？**

- 研究团队希望使开放式对话系统能够理解对话上下文并通过引入相关实体和属性来做出响应。

- 为此，他们使用大型**知识图**（100K实体和1.1M事实）和**数据驱动的推理模型**，该**模型**将对话框转换映射到KG路径，以识别将在特定对话框中提及的实体子集上下文。

- 特别是，研究小组提出了一种名为

  DialKG Walker

>   的新模型，其中包括：

Reviewer: ![image](https://user-images.githubusercontent.com/20548053/70859361-1f856480-1f4d-11ea-9659-15a60906c832.png)
```
特别是，研究小组提出了一种名为**DialKG Walker**的新模型，其中包括：
```

 

#### **人工智能界对此有何看法？**

- 该论文在NAACL-HLT 2019上发表，NAACL-HLT是自然语言处理领域最重要的会议之一。

 

#### **未来的研究领域是什么？**

- 用一些实例指导潜伏模式以选择特定的人格特质。

 

#### **有哪些可能的业务应用程序？**

- 如本文所述，学习说话者之间的个体差异可以被整合到不同的NLP系统中，以提高他们在语言理解，语言生成，人机对话，查询完成等方面的表现。

 

#### **在哪里可以获得实现代码？**

- 作者在[GitHub上](https://github.com/hao-cheng/dynamic_speaker_model)提供了实现代码。

 

### **6.** [**通过开放域对话系统的自演近似交互式人类评估**](https://arxiv.org/abs/1906.09308)，BY ASMA GHANDEHARIOUN, JUDY HANWEN SHEN, NATASHA JAQUES, CRAIG FERGUSON, NOAH JONES, AGATA LAPEDRIZA, ROSALIND PICARD

 

#### **原始摘要**

建立一个开放域的对话代理是一个具有挑战性的问题。当前的评估方法（主要是事后评估的单回合评估）无法在现实的交互式环境中捕获对话质量。在本文中，我们研究了交互式的人工评估并提供了必要的证据。然后，我们介绍一种新颖的，与模型无关且与数据集无关的方法来对其进行近似。特别是，我们提出了一种自演场景，其中对话系统与自己对话，并计算对话轨迹上诸如情感和语义一致性之类的代理的组合。我们表明，该度量标准比迄今为止已知的任何自动化度量标准都能更好地捕获对话模型的人工评估质量，从而实现了显着的Pearson相关性（r> .7，p <.05）。为了研究这种新颖的度量和交互式评估与最新度量和单次评估相比的优势，我们使用一组模型进行了扩展实验，其中包括一些对最近的分层对话框生成体系结构进行了新颖改进的模型通过话语层面的情感和语义知识提炼。最后，我们开源构建的交互式评估平台和收集的数据集，以使研究人员能够有效地部署和评估生成对话模型。包括通过话语级别的情感和语义知识提炼对最近的分层对话框生成体系结构进行新颖改进的几种方法。最后，我们开源构建的交互式评估平台和收集的数据集，以使研究人员能够有效地部署和评估生成对话模型。包括通过话语级别的情感和语义知识提炼对最近的分层对话框生成体系结构进行新颖改进的几种方法。最后，我们开源构建的交互式评估平台和收集的数据集，以使研究人员能够有效地部署和评估生成对话模型。

 

#### **我们的总结**

> 麻省理工学院的研究小组调查了评估开放域对话系统的问题。人是评估对话系统质量的最终权威，但是获得人的评级通常是相当昂贵且困难的过程。为了克服这些挑战并仍然获得可靠的评估，麻省理工学院的团队引入了一种新颖的框架来评估对话质量得分，该得分与人类评级具有很高的统计意义上的相关性。具体来说，他们提出了一系列基于心理学的指标，然后在给定这些指标的情况下拟合了一种功能，以预测人类对对话质量的评估。机器人的质量通过固定轮次中的自打进行评估，其中机器人会生成发声，并在下一轮中将其反馈为输入。实验证实，引入的自玩框架，

Reviewer: ```suggestion
麻省理工学院的研究小组调查了评估开放域对话系统的问题。人是评估对话系统质量的最终权威，但是获得人的评级通常是相当昂贵且困难的过程。为了克服这些挑战并仍然获得可靠的评估，麻省理工学院的团队引入了一种新颖的框架来评估对话质量得分，该得分与人类评级具有很高的统计意义上的相关性。具体来说，他们提出了一系列基于心理学的指标，然后在给定这些指标的情况下拟合了一种功能，以预测人类对对话质量的评估。机器人的质量通过固定轮次中的自打进行评估，其中机器人会生成发声，并在下一轮中将其反馈为输入。实验证实，引入的self-play框架以及基于心理学的自动度量标准可以为会话评估提供良好的智能体。
```

地址： https://www.topbots.com/top-conversational-ai-research-papers-2019/ 

作者： Mariya Yao 

译者：Qiang He

# 2019年对话式AI的10篇重要研究论文

### ![2019年对话式AI的10篇重要研究论文](img/cover_TOP10ConvAI_1600px_web-1280x640.jpg)

会话式AI已成为跨行业业务实践的组成部分。越来越多的公司正在利用聊天机器人为客户服务，销售和营销带来的优势。

即使聊天机器人已成为领先企业的“必备”资产，但其性能仍然与人类相去甚远。来自主要研究机构和技术领导者的研究人员已经探索了各种方法，可以通过增加响应的多样性，实现情感识别，提高其跟踪对话的长期能力，确保保持一致的角色来提高对话系统的性能。等

我们搜索了2019年发表的重要的对话式AI研究论文，向您介绍了在面向任务和开放域对话系统中都设置了最新技术的前10名。

## 2019年对话式AI的10篇重要研究论文  

###  1. [**面向任务的对话系统的可转移多域状态生成器**](https://arxiv.org/abs/1905.08743)， **BY CHIEN-SHENG WU, ANDREA MADOTTO, EHSAN HOSSEINI-ASL, CAIMING XIONG, RICHARD SOCHER, PASCALE FUNG**

#### **原始摘要**

>  对域本体的过度依赖和跨域知识共享的缺乏是对话状态跟踪的两个实际但尚未研究的问题。现有方法通常在推理期间跟踪未知时隙值方面是不足的，并且通常难以适应新的域。在本文中，我们提出了一种可转移的对话状态生成器（TRADE），该函数使用复制机制从发声中生成对话状态，从而在预测训练期间未遇到的（三元组，时段，值）三元组时促进知识的传递。我们的模型由发声编码器，插槽门和状态生成器组成，它们在域之间共享。经验结果表明，对于人与人对话数据集MultiWOZ的五个域，TRADE的最新联合目标准确度达到48.62％。此外，我们通过模拟零镜头和少镜头对话状态跟踪（针对看不见的域）来显示其传输能力。TRADE在零射门域之一中达到了60.58％的联合目标精度，并且能够适应少射门的情况而不会忘记已经受过训练的域。 

Reviewer: ```suggestion
 对域本体的过度依赖和跨域知识共享的缺乏是对话状态跟踪的两个实际但尚未研究的问题。现有方法通常在推理期间跟踪未知时隙值方面是不足的，并且通常难以适应新的域。在本文中，我们提出了一种可转移的对话状态生成器（TRADE），该函数使用复制机制从发声中生成对话状态，从而在预测训练期间未遇到的（三元组，时段，值）三元组时促进知识的传递。我们的模型由发声编码器，插槽门和状态生成器组成，它们在域之间共享。经验结果表明，对于人与人对话数据集MultiWOZ的五个域，TRADE的最新联合目标准确度达到48.62％。此外，我们通过模拟zero-shot和 few-shot对话状态跟踪（针对看不见的域）来显示其传输能力。TRADE在zero-shot域之一中达到了60.58％的联合目标精度，并且能够适应 few-shot的情况而不会忘记已经受过训练的域。 
```

#### **我们的总结**

>  香港科技大学和Salesforce Research的研究小组解决了过度依赖领域本体和缺乏跨领域知识共享的问题。在实际情况中，许多时间段在不同的域之间共享所有或部分值（例如，*区域*时间段可以存在于诸如*餐厅*，*酒店*或*出租车的*许多域中），因此跨多个域传输知识对于对话状态跟踪势在必行（DST）模型。研究人员介绍**TRA** nsferable **d** ialogue统计**Ë**生成器（TRADE），它利用上下文增强的插槽门和复制机制来跟踪对话历史记录中任何地方提到的插槽值。TRADE跨域共享其参数，并且不需要预定义的本体，从而可以跟踪以前看不到的广告位值。实验证明了该方法的有效性，在具有挑战性的MultiWOZ数据集上，TRADE实现了48.62％的最新联合目标精度。 

Reviewer: ```suggestion
香港科技大学和Salesforce Research的研究小组解决了过度依赖领域本体和缺乏跨领域知识共享的问题。在实际情况中，许多时间段在不同的域之间共享所有或部分值（例如，*区域*时间段可以存在于诸如*餐厅*，*酒店*或*出租车的*许多域中），因此跨多个域传输知识对于对话状态跟踪势在必行（DST）模型。研究人员介绍**TRA** nsferable **d** ialogue统计**Ë**生成器（TRADE），它利用上下文增强的插槽门和复制机制来跟踪对话历史记录中任何地方提到的插槽值。TRADE跨域共享其参数，并且不需要预定义的本体，从而可以跟踪以前看不到的广告位值。实验证明了该方法的有效性，在具有挑战性的MultiWOZ数据集上，TRADE实现了48.62％的最新联合目标精度。 
```

 ![对话式AI-贸易](img/1_Dialogue_history_600px_web.jpg) 

#### **本文的核心思想是什么？**

- 为了克服对领域本体的过度依赖和缺乏跨领域知识共享，研究人员建议：

  - 直接生成时隙值，而不是预测每个预定义的本体项的概率；
  - 跨域共享所有模型参数。

- 该贸易模式由三个部分组成：

  - 一个**发声编码器**到编码对话的话语成固定长度的向量序列;
- 一个**槽栅**来预测某一是否*（域，时隙）*对由所述对话触发;
  - 一个**状态生成器**，可独立解码所有*（域，时隙）*对的多个输出令牌，以预测其对应值。

 

![对话式AI-贸易](img/1_TRADE_architecture_800px_web.jpg)

 

#### **关键成就是什么？**

- 在具有挑战性的人与人对话的MultiWOZ数据集上，TRADE的联合目标准确度达到48.62％，开创了新的技术水平。

> - 此外，TRADE在零击域中的一个域中达到了60.58％的联合目标精度，证明了其将知识转移到以前未曾见过的域的能力。

Reviewer: ```suggestion
- 此外，TRADE在zero-shot中的一个域中达到了60.58％的联合目标精度，证明了其拥有将知识转移到以前未曾见过的域的能力。
```

> - 实验还证明了该模型能够适应新的少量快照域而不会忘记已经训练好的域的能力。

Reviewer: ```suggestion
- 实验还证明了该模型能够适应新的few-shot域而不会忘记已经训练好的域的能力。
```

 

#### **人工智能界对此有何看法？**

- 该论文在ACL 2019主会议上获得了杰出论文奖，并在同一会议上获得了NLP的会话人工智能研讨会最佳论文奖。

 

#### **未来的研究领域是什么？**

> - 从其他资源转移知识以进一步提高零击性能。

Reviewer: ```suggestion
- 从其他资源转移知识以进一步提高zero-shot性能。
```

> - 收集具有大量域的数据集，以促进对多域对话状态跟踪中的技术的研究。

Reviewer: ```suggestion
- 收集了具有大量域的数据集，以促进对多域对话状态跟踪中的技术的研究。
```

 

#### **有哪些可能的业务应用程序？**

> - 当前的研究可以在多域设置中显着提高面向任务的对话系统的性能。

Reviewer: ```suggestion
- 当前的研究可以在多域设置中显著提高面向任务的对话系统的性能。
```

 

#### **在哪里可以获得实现代码？**

- 这项研究的PyTorch实现可在[GitHub](https://github.com/jasonwu0731/trade-dst)上[找到](https://github.com/jasonwu0731/trade-dst)。

 

> ### 2. [**神经对话系统可以有效地使用对话历史吗？实证研究**](https://arxiv.org/abs/1906.01603)**，By CHINNADHURAI SANKAR，SANDEEP SUBRAMANIAN，CHRIS PAL，SARATH CHANDAR和YOSHUA BENGIO**

Reviewer: ```suggestion
### 2. [神经对话系统可以有效地使用对话历史吗？实证研究](https://arxiv.org/abs/1906.01603), BY CHINNADHURAI SANKAR, SANDEEP SUBRAMANIAN, CHRIS PAL, SARATH CHANDAR, AND YOSHUA BENGIO
```

 

#### **原始摘要**

> 当建立对话代理时，神经生成模型变得越来越流行。它们提供了灵活性，可以轻松地适应新域，并且需要最少的域工程。对这些系统的普遍批评是，它们很少有效地理解或使用可用的对话历史记录。在本文中，我们通过经验方法来研究这些模型在测试时人为地引入非自然变化或扰动其上下文的敏感性，从而了解这些模型如何使用可用的对话历史记录。我们在4个多转对话数据集上对10种不同类型的扰动进行了实验，发现常用的神经对话架构（例如递归和基于转换器的seq2seq模型）对大多数扰动（例如遗漏或重新排列发音，混排词，

Reviewer: ```suggestion
当建立对话代理时，神经生成模型变得越来越流行。它们提供了灵活性，可以轻松地适应新域，并且需要最少的域工程。对这些系统的普遍批评是，它们很少有效地理解或使用可用的对话历史记录。在本文中，我们通过经验方法来研究这些模型在测试时人为地引入非自然变化或扰动其上下文的敏感性，从而了解这些模型如何使用可用的对话历史记录。我们在4个multi-turn对话数据集上对10种不同类型的扰动进行了实验，发现常用的神经对话体系结构（例如循环神经网络和基于变压器的seq2seq模型）对大多数扰动（例如遗漏或重新排列发音，混排词等）几乎不敏感。通过开源代码，我们相信它将成为将来评估对话框系统的有用诊断工具。
```

 

#### **我们的总结**

> 最近引入的生成模型擅长产生流畅的响应，但是这些响应往往很无聊且重复，这通常归因于这些模型对对话历史的理解不足。在本文中，作者试图通过经验研究神经生成模型是否有效地利用了对话历史。为此，他们测试了递归网络和基于变压器的序列到序列模型对各种合成扰动的敏感性。该方法基于以下假设：如果模型对破坏某些类型信息的扰动不敏感，则意味着该模型将最大限度地利用此信息。实验表明，神经生成模型对发声级和词级摄动不敏感。

Reviewer: ```suggestion
最近引入的生成模型擅长产生流畅的答案，但是这些答案往往很无聊且重复，这通常归因于这些模型对对话历史的理解不足。在本文中，作者试图通过经验研究神经生成模型是否有效地利用了对话历史。为此，他们测试了循环网络和基于transformer的seq-to-seq模型对各种合成扰动的敏感性。该方法基于以下假设：如果模型对破坏某些类型信息的扰动不敏感，则意味着该模型将最大限度地利用此信息。实验表明，神经生成模型对发声级和词级摄动不敏感。
```

 

![对话式AI研究](img/2_Shuffling_sample_800px_web.jpg)

 

#### **本文的核心思想是什么？**

- 在对对话历史进行综合引入扰动的情况下，以经验方式研究生成神经系统的行为。
- 实验：
  - **话语水平扰动**（搅乱话语序列、颠倒话语顺序、删除某些话语、截断对话历史以仅包含k个最近的话语）;
  - **单词级扰动**（单词洗牌，颠倒单词顺序，均匀地减少30%的单词，删除所有名词，删除所有动词)。

 

#### **关键成就是什么？**

- 凭经验证明：
  - 即使在对话历史记录发生极端变化的情况下，模型也往往只会显示出微小的困惑，这表明它们没有有效地利用可用的信息。
  - Transformer 对单词重排不敏感，这意味着它们可以学习类似单词袋的表示形式。
  - 注意机制导致模型使用对话框前面部分中的更多信息。
  - 与递归模型相比，Transformer 对扰动不敏感，扰动会破坏发声间的对话动态。

 

#### **人工智能界对此有何看法？**

- 该论文已被提名为ACL 2019最佳论文奖的候选人。

 

#### **未来的研究领域是什么？**

- 模型对扰动的敏感性会破坏对话框历史记录中的某些类型的信息，可能会很有用：
  - 用于了解模型用于解决新对话框数据集的信息种类；
  - 作为新模型的诊断工具。

 

#### **在哪里可以获得实现代码？**

- 该代码可在[GitHub上获得](https://github.com/chinnadhurai/ParlAI/)。

 

> ### **3.** [**善于说服：建立个人的有说服力的社会公益**说服对话系统](https://arxiv.org/abs/1906.06725)，BY CHINNADHURAI SANKAR, SANDEEP SUBRAMANIAN, CHRIS PAL, SARATH CHANDAR, AND YOSHUA BENGIO

Reviewer: ```suggestion
### 3.[善于说服：建立个人的有说服力的社会公益说服对话系统](https://arxiv.org/abs/1906.06725)，BY CHINNADHURAI SANKAR, SANDEEP SUBRAMANIAN, CHRIS PAL, SARATH CHANDAR, AND YOSHUA BENGIO
```

 

#### **原始摘要**

> 开发智能的有说服力的对话代理程序以改变人们的意见和行为以实现社会利益，是推进自动化对话系统的道德发展的前沿。为此，第一步是要了解在人的说服性对话中运用的战略披露和申诉的复杂组织。我们设计了一个在线说服任务，要求一位参与者说服另一位参与者向特定的慈善机构捐款。我们收集了包含1,017个对话的大型数据集，并从一个子集中注释了新兴的说服策略。基于注释，我们构建了带有上下文信息和句子级功能的基线分类器，以预测语料库中使用的10种说服策略。此外，为了了解个性化的说服流程，我们分析了个人的人口统计学和心理背景之间的关系，包括人格，道德，价值体系及其捐赠意愿。然后，我们根据个人的个人背景分析了哪种类型的说服策略导致了更多的捐赠。这项工作为开发个性化的说服性对话系统奠定了基础。

Reviewer: ```suggestion
开发智能的有说服力的对话智能体程序以改变人们的意见和行为以实现社会利益，是推进自动化对话系统的道德发展的前沿。为此，第一步是要了解在人的说服性对话中运用的战略披露和申诉的复杂组织。我们设计了一个在线说服任务，要求一位参与者说服另一位参与者向特定的慈善机构捐款。我们收集了包含1,017个对话的大型数据集，并从一个子集中注释了新兴的说服策略。基于注释，我们构建了带有上下文信息和句子级功能的基线分类器，以预测语料库中使用的10种说服策略。此外，为了了解个性化的说服流程，我们分析了个人的人口统计学和心理背景之间的关系，包括人格，道德，价值体系及其捐赠意愿。然后，我们根据个人的个人背景分析了哪种类型的说服策略导致了更多的捐赠。这项工作为开发个性化的说服性对话系统奠定了基础。
```

 

#### **我们的总结**

本文建立在细化可能性模型的基础上，该模型认为，有说服力的信息根据人们的世界观而量身定制时会更有效。作者从Mechanical Turk招募了参与者，从心理上对他们进行了介绍，然后请他们扮演角色，说服对方，向慈善机构Save the Children捐款。然后，研究人员根据所使用的说服策略对会话的一部分进行注释，并建立了一个混合RCNN分类器，将整个语料库分为不同的说服策略。他们分析了总体上最成功的策略以及说服策略与用户的人口统计特征和性格类型之间的相互作用。

 

![会话式AI-说服数据集](img/3_Persuasion_dataset_500px_web.jpg)

 

#### **本文的核心思想是什么？**

> - AI对话代理的目的通常涉及某种形式的说服，但是对他们有效的说服策略的研究受到限制，因为说服的研究传统上是社会科学而不是AI工程的一部分。

Reviewer: ```suggestion
- AI对话智能体的目的通常涉及某种形式的说服，但是对他们有效的说服策略的研究受到限制，因为说服的研究传统上是社会科学而不是AI工程的一部分。
```

- 这项跨学科的论文建立在社会学基础上，可以在大量的人类聊天对话中识别不同的说服策略，并分析哪种策略对一般人和给定的性格类型最有效。
- 为了将说服策略分为10类，再加上一类“非策略”类，作者开发了具有以下特征的混合RCNN模型：
  - 句子嵌入

>   - 上下文嵌入；

Reviewer: ```suggestion
  - 上下文嵌入
```

>   - 句子级功能。

Reviewer: ```suggestion
  - 句子级功能
```

 

![对话式AI-说服力](img/3_Persuastion_classifier_500px_web.jpg)

*混合RCNN分类器概述*

 

#### **关键成就是什么？**

- 研究人员发现：
  - 提供实用的捐赠信息是总体上最好的策略。
  - 向用户询问他们是否熟悉慈善机构，大大提高了对开放度五大特征的参与者的捐赠可能性。
  - 提出个人问题会大大提高订阅者对“关爱”或“自由”的捐赠概率（按照海特的道德基础理论），但会降低对订阅“公平或权威”的捐赠者的捐赠概率。

 

#### **人工智能界对此有何看法？**

- 该论文已被提名为ACL 2019最佳论文奖的候选人。

 

#### **未来的研究领域是什么？**

- 通过包含更多注释和更多对话框上下文来提高分类器的性能。
- 设计一种自适应的说服对话系统，该系统具有根据用户个人资料选择说服策略的能力。

 

#### **有哪些可能的业务应用程序？**

- 作者指出，技术是双刃剑，并具有说服人们善与恶的潜力。牢记这一点，本文中介绍的知识可用于为自动对话系统的道德设计基础提供信息。
- 按照本文中的示例，慈善机构可以使用它说服潜在的支持者捐款。
- 它也可以用于旨在帮助人们实现其个人目标的产品中。

 

#### **在哪里可以获得实现代码？**

- 作者已经在[GitLab上](https://gitlab.com/ucdavisnlp/persuasionforgood)发布了数据集和代码。

 

> ### **4.** [**OPENDIALKG：基于注意力的知识图遍历的可解释性**](https://www.semanticscholar.org/paper/OpenDialKG%3A-Explainable-Conversational-Reasoning-Moon-Shah/0d3c68c207fc83fb402b7217811af22066300fc9)[**对话推理**](https://www.semanticscholar.org/paper/OpenDialKG%3A-Explainable-Conversational-Reasoning-Moon-Shah/0d3c68c207fc83fb402b7217811af22066300fc9)，By SEUNGWHAN MOON，PARARTH SHAH，ANUJ KUMAR和RAJEN SUBBA

Reviewer: ```suggestion
### 4. [OPENDIALKG：基于注意力的知识图遍历的可解释性对话推理](https://www.semanticscholar.org/paper/OpenDialKG%3A-Explainable-Conversational-Reasoning-Moon-Shah/0d3c68c207fc83fb402b7217811af22066300fc9)，By SEUNGWHAN MOON，PARARTH SHAH，ANUJ KUMAR和RAJEN SUBBA
```

 

#### **原始摘要**

> 我们研究了一种对话推理模型，该模型策略性地遍历了大型公共事实知识图（KG），以引入引人入胜且在上下文中不同的实体和属性。在这项研究中，我们收集了一个新的**开放**式**Dial** og **KG**平行语料库，称为**OpenDialKG**，其中来自人与人角色扮演对话的15K的每个讲话都用地面真实引用手动注释，这些引用来自具有1M +事实的大型KG的相应实体和路径。然后，我们提出DialKG Walker模型，该模型学习对话上下文作为KG上的结构化遍历的对话上下文的符号转换，并预测自然实体，通过新颖的领域无关，基于注意力的图路径解码器引入给定的先前对话上下文。自动和人工评估表明，在域内和跨域任务中，我们的模型都可以比最新的基准或基于规则的模型检索更多自然和类似人类的响应。所提出的模型还为检索到的每个实体生成了KG行走路径，从而提供了一种解释对话推理的自然方法。

Reviewer: ```suggestion
我们研究了一种对话推理模型，该模型策略性地遍历了大型公共事实知识图（KG），以引入让人感兴趣且在上下文中不同的实体和属性。在这项研究中，我们收集了一个新的**Open-ended Dialog**↔ **KG**平行语料库，称为**OpenDialKG**，其中来自人与人角色扮演对话的15K的每个讲话都用地面真实引用手动注释，这些引用来自具有1M +事实的大型KG的相应实体和路径。然后，我们提出DialKG Walker模型，该模型学习对话上下文作为KG上的结构化遍历的对话上下文的符号转换，并预测自然实体，通过新颖的领域无关，基于注意力的图路径解码器引入给定的先前对话上下文。自动和人工评估表明，在域内和跨域任务中，我们的模型都可以比最新的基准或基于规则的模型检索更多自然和类似人类的响应。所提出的模型还为检索到的每个实体生成了KG行走路径，从而提供了一种解释对话推理的自然方法。
```

 

#### **我们的总结**

Facebook对话式AI团队采用了一种新颖的方法来创建自然的，类似于人的反应，并提供有关不同实体及其属性的引人入胜的，上下文相关的信息。首先，他们收集一个新的并行语料库**OpenDialKG**，在对话中对实体的每次提及都与其相应的真实KG路径手动链接。然后，作者介绍了一种称为**DialKG Walker**的新模型，该模型可以学习对话中提到的实体之间的知识路径以及基于大型常识知识图的原因。在多个基准上进行的实验表明，与最新的基准相比，建议的方法可产生更多自然和类似人的反应。

 

![OpenDialKG对话式AI](img/4_OpenDialKG_overview_550px_web.jpg)

 

#### **本文的核心思想是什么？**

- 研究团队希望使开放式对话系统能够理解对话上下文并通过引入相关实体和属性来做出响应。

- 为此，他们使用大型**知识图**（100K实体和1.1M事实）和**数据驱动的推理模型**，该**模型**将对话框转换映射到KG路径，以识别将在特定对话框中提及的实体子集上下文。

- 特别是，研究小组提出了一种名为

  DialKG Walker

  的新模型，其中包括：

  - 一个基于注意力的**图解码器**，它在大型常识性KG中走出一条最佳路径，以有效地修剪不太可能的候选实体；

>   - 一个并行的**零击学习模型**，该**模型**利用先前的句子，对话和KG上下文根据候选实体的相关性和路径得分对其进行排名。

Reviewer: ```suggestion
  - 一个并行的**zero-shot 学习模型**，该**模型**利用先前的句子，对话和KG上下文根据候选实体的相关性和路径得分对其进行排名。
```

- 借助大规模的公共事实知识图，引入的方法可以在跨各个域和任务的开放式对话中实现与域无关的对话推理。

 

![OpenDialKG架构](img/4_OpenDialKG_architecture_800px_web.jpg)

 

#### **关键成就是什么？**

- 自动和人工评估表明DialKG Walker优于最新的基准和基于规则的模型。
- 广泛的跨域和转移学习评估证实了所提出方法的灵活性。
- 引入的OpenDialKG数据集在15K对话会话中具有91K语音，可以深入研究符号推理和自然语言对话。

 

#### **人工智能界对此有何看法？**

- 该论文已被提名为ACL 2019最佳论文奖的候选人。

 

#### **有哪些可能的业务应用程序？**

> - 通过提及相关实体和属性，使跨域和任务（例如，聊天，建议）的聊天机器人能够生成更自然和更具吸引力的响应。

Reviewer: ```suggestion
- 通过提及相关实体和属性，使跨域和任务（例如，聊天，建议）的聊天机器人能够生成更自然和更具吸引力的回答。
```

> ### **5.** [**对话互动的动态说话人模型**](https://www.aclweb.org/anthology/N19-1284)，BY HAO CHENG, HAO FANG, MARI OSTENDORF

Reviewer: ```suggestion
### 5. [对话互动的动态说话人模型](https://www.aclweb.org/anthology/N19-1284)，BY HAO CHENG, HAO FANG, MARI OSTENDORF
```

 

#### **原始摘要**

> 说话者的个体差异反映在他们的语言使用以及他们的兴趣和观点上。表征这些差异对于人机交互以及人与人之间的对话分析很有用。在这项工作中，我们介绍了一种神经模型，用于学习在对话上下文中嵌入动态更新的说话者。初始模型训练是无监督的，使用上下文相关的语言生成作为目标，而上下文是对话历史记录。进一步的微调可以利用与任务有关的监督培训。研究表明，学习者对说话人的神经表示对于社交机器人中的内容排名和人与人对话中的对话行为预测非常有用。

Reviewer: ```suggestion
说话者的个体差异反映在他们的语言使用以及他们的兴趣和观点上。表示这些差异对于人机交互以及人与人之间的对话分析很有用。在这项工作中，我们介绍了一种神经模型，用于学习在对话上下文中嵌入动态更新的说话者。初始模型训练是无监督的，使用上下文相关的语言生成作为目标，而上下文是对话历史记录。进一步的微调可以利用与任务有关的监督培训。研究表明，学习者对说话人的神经表示对于社交机器人中的内容排名和人与人对话中的对话行为预测非常有用。
```

 

#### **我们的总结**

> 华盛顿大学的研究小组建议，了解用户之间的个体差异对预测下一个对话行为很有用。为了发现这些个体差异，研究人员建议使用无监督的神经模型，该模型从对话历史中学习说话者嵌入。此外，为了捕获随时间的变化并在有新数据可用时改进说话者表示，引入的模型的结构允许在每次对话时动态更新说话者嵌入向量。实验结果表明，具有动态说话者嵌入功能的模型在预测人与人之间的机器人对话中的用户主题决策以及对人与人对话中的对话行为进行分类时，优于基线。

Reviewer: ```suggestion
华盛顿大学的研究小组建议，了解用户之间的个体差异对预测下一个对话行为很有用。为了发现这些个体差异，研究人员建议使用无监督的神经模型，该模型从对话历史中学习说话者嵌入。此外，为了捕获随时间的变化并在有新数据可用时改进说话者表征，引入的模型的结构允许在每次对话时动态更新说话者嵌入向量。实验结果表明，具有动态说话者嵌入功能的模型在预测人与人之间的机器人对话中的用户主题决策以及对人与人对话中的对话行为进行分类时，优于基线。
```

 

![动态扬声器](img/5_Dynamic_speaker_model_800px_web.jpg)

*动态扬声器模型*

 

#### **本文的核心思想是什么？**

- 作者介绍了一种用于学习说话者神经表示的模型：

>   - 该模型以无监督的方式进行训练，以仅通过依靠该说话者的对话历史来学习每个说话者的表示。

Reviewer: ```suggestion
  - 该模型以无监督的方式进行训练，以仅通过依靠该说话者的对话历史来学习每个说话者的表征。
```

  - 将用于分析说话者潜在模式的可学习组件整合到模型中，以帮助使说话者的学习特征与人类可解释的特征保持一致。

- 该

  动态音箱型号

>   由三个部分组成：

Reviewer: ![image](https://user-images.githubusercontent.com/20548053/70858035-c3aee180-1f34-11ea-94c5-e00ef8522d56.png)
这部分改成
```
该**动态说话者模型**由三个部分组成
```
